### ISSUE TITLE
Getting Started > PHP > CRUD Operations [JIRA: DOC-711]

### ISSUE THREAD

**Issue by pogzie** [basho/basho_docs#2289]
In the section Reading Objects From Riak (http://docs.basho.com/riak/kv/2.1.4/developing/getting-started/php/crud-operations/#reading-objects-from-riak) the sample with the `assert()` function does not output anything. 

Current test machine: 
```
root@vagrant-ubuntu-trusty-64:/home/vagrant# lsb_release -d
Description:	Ubuntu 14.04.4 LTS
root@vagrant-ubuntu-trusty-64:/home/vagrant# php --version
PHP 5.5.9-1ubuntu4.20 (cli) (built: Oct  3 2016 13:00:37) 
Copyright (c) 1997-2014 The PHP Group
Zend Engine v2.5.0, Copyright (c) 1998-2014 Zend Technologies
    with Zend OPcache v7.0.3, Copyright (c) 1999-2014, by Zend Technologies
```

I opted to use the `print_r()` function instead. Changed to:

```
print_r($response1->getObject()->getData());
print_r($response2->getObject()->getData());
print_r($response3->getObject()->getData());
```

And gave a more expected output:

```
root@vagrant-ubuntu-trusty-64:~/riak-client-tester/php# php riak.php
1twoArray
(
    [myValue] => 3
)

```

**Comment by laurenrother** [comment]
Fixed by PR #2307

**Comment by Basho-JIRA** [comment]
Fixed by https://github.com/basho/basho_docs/pull/2307 

***[posted via JIRA by Lauren Rother]***
